config/config/wc-config.yaml

```diff
@@ -252,3 +255,18 @@ networkPolicies:
 
   alertmanager:
     enabled: true
+
+## Open policy agent configuration
+gatekeeper:
+  restrictUserCRDs:
+    enabled: false
+    enforcement: deny
+
+  sealedSecrets:
+    enabled: false
+
+  mongodb:
+    enabled: false
```

Comment:
I'm questioning if these should be under gatekeeper or somewhere under user. Since they mostly (only?) add rbac.

Reply:
Are you considering just the sealedSecrets/mongodb part or he complete gatekeeper part?

Reply:
I was thinking just the sealedSecrets/mongodb part